stdlib/collections/__init__.pyi

```diff
@@ -328,15 +328,15 @@ class _OrderedDictValuesView(ValuesView[_VT_co], Reversible[_VT_co]):
 # (At runtime, these are called `odict_keys`, `odict_items` and `odict_values`,
 # but they are not exposed anywhere)
 @final
-class _odict_keys(dict_keys[_KT_co, _VT_co], Reversible[_KT_co]):  # type: ignore[misc]
+class _odict_keys(dict_keys[_KT_co, _VT_co], Reversible[_KT_co]):  # type: ignore[misc]  # pyright: ignore
```

Comment:
Are we able to add the pyright error codes for these `pyright: ignore`s?

Reply:
There is no code!
> Base class "dict_keys[_KT_co@_odict_keys, _VT_co@_odict_keys]" is marked final and cannot be subclassed `Pylance`

> E:\Users\Avasam\Documents\Git\typeshed\stdlib\collections\__init__.pyi
    E:\Users\Avasam\Documents\Git\typeshed\stdlib\collections\__init__.pyi:331:19 - error: Base class "dict_keys[_KT_co@_odict_keys, _VT_co@_odict_keys]" is marked final and cannot be subclassed

Reply:
Oh, strange! In that case, can we leave a comment to that effect? (Same below for the other one)

```suggestion
# pyright doesn't have a specific error code for this error!
class _odict_keys(dict_keys[_KT_co, _VT_co], Reversible[_KT_co]):  # type: ignore[misc]  # pyright: ignore
```